packages/base/links-to-editor.gts

```diff
@@ -144,11 +143,12 @@ class LinksToEditor extends GlimmerComponent<Signature> {
 export function getLinksToEditor(
   model: Box<CardDef | null>,
   field: Field<typeof CardDef>,
-  context?: CardContext,
 ): ComponentLike<{ Args: {}; Blocks: {} }> {
   return class LinksToEditTemplate extends GlimmerComponent {
     <template>
-      <LinksToEditor @model={{model}} @field={{field}} @context={{context}} />
+      <CardContextConsumer as |context|>
+        <LinksToEditor @model={{model}} @field={{field}} @context={{context}} />
```

Comment:
we should consume context via @consume in LinksToEditor instead of passing it here